WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -175,7 +175,7 @@ final class EditableOrderViewModel: ObservableObject {
 
     /// Product Variations list
     ///
-    private var allProductVariations: [ProductVariation] = []
+    private var allProductVariations: Set<ProductVariation> = []
```

Comment:
This is a great improvement. Interesting to see that we already had Products conform to hashable, but not variations.

Reply:
Yeah, thanks 👍